lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2000,6 +2004,26 @@ def container_stop(self, container_id):
         return response
 
 
+class AnsibleDockerClientContainer(AnsibleDockerClient):
+
+    def __init__(self, **kwargs):
+        super(AnsibleDockerClientContainer, self).__init__(**kwargs)
+
+        docker_api_version = self.version()['ApiVersion']
+        init_supported = LooseVersion(docker_api_version) < LooseVersion('1.25')
```

Comment:
Shouldn't this be a `>` test?

Reply:
(Actually, `>=`.)

Reply:
Can confirm, the tests should be `>=`. With those modifications, I have sucessfully applied this patch to v2.6.0 (8ce3bd6dea).